src/System.Private.ServiceModel/src/Resources/Strings.resx

```diff
@@ -3108,51 +2964,24 @@
   <data name="UnknownSequenceFaultReason" xml:space="preserve">
     <value>The value of wsrm:Identifier is not a known Sequence identifier.</value>
   </data>
```

Comment:
The "Sequence" and "RM" messages above can probably also be removed at some point